src/Examples/AlexNet.cs

```diff
@@ -58,6 +58,17 @@ public override Tensor forward(Tensor input)
             using (var x = avg.reshape(new long[] { avg.shape[0], 256 * 2 * 2 }))
                 return classifier.forward(x);
         }
+
+        protected override void Dispose(bool disposing)
+        {
+            if (disposing) {
```

Comment:
Newer versions of the C# compiler will complain if you never call Dispose() on submodules. Even if most of these instances will live for as long as the process, this is a code cleanliness matter.